Feature: Simple Calculator Program

Yui089 opened the pull request:
This pull request adds the Simple Calculator Program under the package `programs.SimpleCalculator`.

The program performs:
- Addition
- Subtraction
- Multiplication
- Division (with zero-check validation)

Author: Haiana Gracia Peregrino
Description: Part of the CC20 GitHub OOP Activity project.

Yui089 committed:
Add Haiana Peregrino program

MichaelPagaran commented:
Please attach a screenshot here as proof that your program is working on your branch. In addition, I need you to write docstrings that includes your name and basic info about the program just like this:
<img width="689" height="310" alt="1" src="https://github.com/user-attachments/assets/350eb7b8-8e40-48c7-9e20-80e3c5a6f343" />

Yui089 committed:
Updated SimpleCalculator description and fixed commments

Yui089 commented:
Updated SimpleCalculator program.
- Added description comments
- Implemented basic arithmetic functionality
<img width="1362" height="821" alt="Screenshot 2025-11-14 163446" src="https://github.com/user-attachments/assets/b2872d8b-32d3-41cc-ad82-c38a161c40f6" />
